src/main/java/com/attacktimer/AttackTimerMetronomeTileOverlay.java

```diff
@@ -34,14 +42,16 @@ public AttackTimerMetronomeTileOverlay(Client client, AttackTimerMetronomeConfig
         this.client = client;
         this.config = config;
         this.plugin = plugin;
-        setPosition(OverlayPosition.DYNAMIC);
-        setLayer(OverlayLayer.UNDER_WIDGETS);
+        setPosition(OverlayPosition.TOP_LEFT);
```

Comment:
This commit looks like it changes the default rendering - let's make sure that the default remains the same and any movability is opt-in via the config interface.

Reply:
I imagine anything that is giving a timer for your attacks you would want overlayed any other rendered objects. 
That being said I can attempt to make it a option in the config.